project_4/styles.css

```diff
@@ -0,0 +1,667 @@
+}
+
+/*  */
+
+.left-friday,
+.right-saturday {
+    margin: auto;
+    width: 80%;
+}
+
+h4 {
+    border-left: 5px var(--color-blue) solid;
+    font-family: var(--font-header);
+    font-size: 20px;
+    padding: 10px;
+    text-transform: uppercase;
+}
+
+/* ================
+REGISTER SECTION
+================== */
+
+.register {
+    background-color: var(--color-green);
+    margin-top: 100px;
+    padding: 100px 30px;
+    text-align: center;
+}
+
+.register a {
+    border: 1px dashed var(--color-offwhite);
+    color: white;
+    font-family: var(--font-header);
+    font-size: 30px;
+    padding: 10px;
+    text-decoration: none;
+    text-transform: uppercase;
+    width: 200px;
+}
+
+.register-text {
+    color: white;
+    font-family: var(--font-header);
+    font-size: 18px;
+    margin-bottom: 40px;
+}
+
+/* hover effects for register button */
+.register a:hover {
+    background-color: var(--color-brown);
+}
+
+/* ===============
+FOOTER
+================= */
+
+.footer {
+    background-color: var(--color-brown);
+    height: 200px;
+    margin-top: 150px;
+    padding: 60px 40px 10px;
+}
+
+/* social media icons */
+.social-media-icons a {
+    display: inline-block;
+    fill: var(--color-blue);
+    padding: 15px;
+    transition: transform .2s;
+    width: 40px;
+}
+
+.social-media-icons a:hover {
+    fill: var(--color-green);
+    transform: scale(1.5);
+}
+
+.follow-us {
+    color: white;
+    max-width: 100%;
+}
+
+.copyright {
+    color: var(--color-offwhite);
+    font-size: 9px;
+}
+
+.follow-us,
+.copyright {
+    font-family: var(--font-paragraph);
+}
+
+
+/* ==============
+MEDIA QUERIES
+================= */
+
+@media (max-width: 420px) {
+    .register a {
+        font-size: 26px;
+    }
+}
+
+@media (min-width: 600px) {
+    .speakers {
+        width: 50%;
+    }
+
+    nav ul a {
+        font-size: 17px;
+        padding: 30px 20px;
+    }
+}
+
+
+@media (min-width: 800px) {
+
+    /* header 2 */
+    .section-header {
+        font-size: 70px;
+    }
+
+    /* navigation */
+    nav ul a {
+        font-size: 17px;
+        padding: 30px 40px;
+    }
+
+    nav h1 a {
+        letter-spacing: 6px;
+    }
+
+    nav ul a::before {
+        color: var(--color-blue);
+        margin-right: 15px;
+        content: "|"
+    }
+
+    nav ul a::after {
+        color: var(--color-blue);
+        margin-left: 15px;
+        content: "|";
+
+    }
+
+    /* header */
+    .home-page-title {
+        font-size: 55px;
+        width: 400px;
+    }
+
+    .home-page-register {
+        padding: 20px 100px;
+    }
+
+    .date-location {
+        font-size: 20px;
+    }
+
+    /* mission section */
+    .quotes {
+        margin: auto;
+        width: 60%;
+        padding: 30px 100px;
+        text-align: center;
+    }
+
+    .left-column,
+    .right-column {
+        display: inline-block;
+        margin: 0 4%;
+        vertical-align: top;
+        width: 40%;
+    }
+
+    .left-column,
+    .right-column {
```

Comment:
This appears to duplicate and directly override the previous rule set on lines 531-537.